transport/wsr/src/test/java/org/kaazing/gateway/transport/wsr/WsrAcceptorTest.java

```diff
@@ -29,12 +29,18 @@
 import java.util.Collections;
```

Comment:
Removed file but #287 also tracks wse bind failures so that has to stay open :-(